periphery/src/BorrowManager.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.15;
+
+import {ERC20, SafeTransferLib} from "solmate/utils/SafeTransferLib.sol";
+import "v3-core/contracts/interfaces/IUniswapV3Pool.sol";
+import {Uniswap} from "aloe-ii-core/libraries/Uniswap.sol";
+import {Factory} from "aloe-ii-core/Factory.sol";
+import {IManager, Borrower} from "aloe-ii-core/Borrower.sol";
+import {Lender} from "aloe-ii-core/Lender.sol";
+
+contract BorrowManager is IManager {
+    using SafeTransferLib for ERC20;
+
+    Factory public immutable FACTORY;
+
+    constructor(Factory _factory) {
+        FACTORY = _factory;
+    }
+
+    /* solhint-disable code-complexity */
+
+    /**
+     * @notice This should be called by a borrower during its modify function
+     * @param data actions, amount0, amount1
```

Comment:
I'd recommend being more specific here. Something like the following (depends on my other comments):

```
@param data Obtained from `abi.encode(actions, amount0, amount1)`. User should pass this in when calling `borrower.modify`
```